tests/test_examples/test_runs_via_configs/test_via_configs.py

```diff
@@ -27,7 +27,9 @@ def run(file: str) -> None:
 
 
 def test_train() -> None:
```

Comment:
In the future we definitely need something to show fail reasons on this `run(...)` tests
Because now it looks like "Exit with code 1" and no additional info provided

Reply:
agree